spec/peep_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'peep'
+
```

Comment:
To get 100% coverage, you'd also need tests for `.all` with filtering and `.toggle_order`